src/modules/Checkbox/Checkbox.js

```diff
@@ -258,6 +258,7 @@ export default class Checkbox extends Component {
           tabIndex={this.computeTabIndex()}
           type={type}
           value={value}
+          onClick={e => e.stopPropagation()}
```

Comment:
Can we create a `handleInputClick` method and make a comment there with a link to the matching issue?

Reply:
And sort props in alphabetic order ❤️

Reply:
Done!